README.md

```diff
@@ -1,4 +1,14 @@
-# Helios Design System
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/1672302/213473448-ac8cd0f7-cde2-49ef-b8ef-4067677569f2.jpg">
```

Comment:
```suggestion
  <img src="https://user-images.githubusercontent.com/1672302/213473448-ac8cd0f7-cde2-49ef-b8ef-4067677569f2.jpg" role="presentation" alt="">
```

Question: Why are we not hosting this image on our own website?

Reply:
> Why are we not hosting this image on our own website

The assets hosted on our website are fingerprinted and therefore would change frequently

Reply:
added the role in a new commit